src/getEdgeMassMatrix.jl

```diff
@@ -1,54 +1,291 @@
-export getEdgeMassMatrix, getdEdgeMassMatrix, getEdgeMassMatrixNoWeight
+export getEdgeMassMatrix, getdEdgeMassMatrix
 
 
+"""
 function getEdgeMassMatrix(M::OcTreeMeshFV,sigma::Vector)
-    if isempty(M.Pe)
-        M.Pe = getEdgeMassMatrixAnisotropic(M.S,M.h)
-    end
-	M = M.Pe' * kron(speye(24),sdiag(sigma)) * M.Pe
-	return M
+
+Returns finite volume edge mass matrix. Can handle isotropic and anisotropic models.
+
+
+Input:
+   M::OcTreeMeshFV Octree mesh object
```

Comment:
very minor comment. I think you want to add an empty line after input so that the docstring is formatted better. Try ?getEdgeMassMatrix if you have a chance...